d3rlpy/base.py

```diff
@@ -304,11 +310,14 @@ def fit(self,
                                       with_timestamp, logdir, verbose,
                                       tensorboard)
 
+        # add reference to active logger to algo class during fit
+        self.active_logger_ = logger
+
         # save hyperparameters
         self._save_params(logger)
 
         # refresh evaluation metrics
-        self.eval_results_ = {}
+        self.eval_results_ = defaultdict(list)
```

Comment:
Probably, we better refresh `loss_history_` here.

Reply:
Done